android-components/components/browser/toolbar/src/main/res/layout/mozac_browser_toolbar_displaytoolbar.xml

```diff
@@ -29,10 +29,12 @@
         android:layout_width="0dp"
```

Comment:
we need to change the margin depending on whether the action button is displayed or not, see details for demo